src/sentry/static/sentry/app/components/searchBar.tsx

```diff
@@ -104,4 +114,8 @@ class SearchBar extends React.PureComponent<Props, State> {
   }
 }
 
+const Input = styled('input')`
+  width: ${p => (p.width ? p.width : undefined)};
```

Comment:
Will this result in `width: undefined`? Perhaps this whole property should be conditional?

Reply:
Emotion seems to be smart enough to omit the thing if the returned value is undefined.
```
const Input = styled('input')`
  width: ${p => p.width ?? undefined};
  /* same result */
  ${p => p.width && `width: ${p.width}`};
`;
```

Note that if you would type undefined explicitly like this, width would be 'undefined' in the final css:
```
const Input = styled('input')`
  width: undefined;
`;
```